doc/setup_instructions.md

```diff
@@ -50,11 +50,19 @@ LFS. Note that the GitHub option to download as .zip will not work
 
 You need at least Git LFS version 2.8.0, old versions do not work.
 
-On Linux use your package manager to install Git. On Mac install the
-package manager [homebrew](https://brew.sh/) if you don't already have
-it, and use it to install Git. On Mac and Linux Git LFS is likely available
-as a package named `git-lfs`. If it is not install it manually. After
-installing remember to run `git lfs install` in terminal.
+On Linux use your package manager to install Git. This can be done in the
+terminal - for instance, on Ubuntu, type: `sudo apt-get install git` . On
+Mac, install the package manager [homebrew](https://brew.sh/) if you don't
+already have it, and use it to install Git. On Mac and Linux Git LFS is
+likely available as a package named `git-lfs`. If it is not, install it
+manually. You can download Git LFS from https://git-lfs.com. After
+downloading the file, extract it wherever you want. In the terminal, change
+directory to the newly extracted folder (e.g. " cd ~/Games/git-lfs-3.4.1 ").
+You can then install it via the terminal - for instance on Ubuntu, type:
+`sudo bash ./install.sh`.
+OR, install Git LFS entirely by typing in the terminal - for instance on
+Ubuntu, type: `git lfs install`. 
+It should then say "Git LFS initialized."
```

Comment:
This last part no longer makes sense to me, because even if you use a package manager to install git LFS, you *have to* run the install command. So if you want to put in the manual install command (which BTW is not distro specific, and should probably be in the form on `sudo ./install.sh` to not force bash to evaluate the file if the file shebang line asks for example just sh) I'd make it much more clear that that part was end of that section. Probably would be clearer to make the git lfs install part a separate paragraph.